specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-01-31-preview/applications.json

```diff
@@ -0,0 +1,338 @@
+          },
+          {
+            "name": "applicationName",
+            "in": "path",
+            "required": true,
+            "description": "name of an application resource",
+            "pattern": "^[^<>%&:?#/\\\\]+$",
+            "minLength": 1,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "./types/applications.json#/definitions/application"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get an Application resource": {
+            "$ref": "./examples/GetAnApplication.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Applications"
+        ],
+        "operationId": "Applications_Update",
+        "description": "Updates an application instance.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./types/applications.json#/definitions/applicationUpdate"
+            },
+            "description": "The update to the application instance."
+          },
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "name": "applicationName",
+            "in": "path",
+            "required": true,
+            "description": "name of an application resource",
+            "pattern": "^[^<>%&:?#/\\\\]+$",
+            "minLength": 1,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated an application instance",
+            "schema": {
+              "$ref": "./types/applications.json#/definitions/application"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update a chaos application instance": {
+            "$ref": "./examples/ApplicationPatchUpdate.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Applications"
+        ],
+        "operationId": "Applications_CreateOrUpdate",
+        "description": "Create a Application",
```

Comment:
>Create [](http://example.com/codeflow?start=24&length=6)

Create or update an application

Reply:
Updated the description for the operation